bin/sidecli.re

```diff
@@ -342,6 +553,8 @@ let () = {
       (create_transaction, info_create_transaction),
       (sign_block, info_sign_block),
       (produce_block, info_produce_block),
+      (gen_credentials, info_gen_credentials),
+      (inject_genesis, info_inject_genesis),
```

Comment:
inject_genesis is not needed, it should be attainable with produce_block + sign_block

Reply:
I feel like it's a useful command just for convenience

Reply:
Inject_genesis should almost never happen, we can have an alias, but not duplicate the logic. And inject_genesis also only makes sense if you control all the keys, which we will not in the future

Reply:
Made an issue for it: #45

Reply:
How can the genesis block be created with `produce-block` command? `produce-block` only takes path to the `state.bin` file right now.

Reply:
@callistonianembrace and it will produce a block, you can run a node to get the state.bin, will move this to an issue to document on how this can be done

#59